echo/server/run-worker.sh

```
@@ -1,45 +1 @@
-launch_worker "worker.cpu" "cpu"
-
-# Launch Flower
-echo "Launching Flower"
-export FLOWER_UNAUTHENTICATED_API=True
-celery -A dembrane.tasks flower &
-FLOWER_PID=$!
-
-# Wait for all background jobs to finish
-wait
\ No newline at end of file
+dramatiq-gevent --queues network --processes 2 --threads 50 dembrane.tasks
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)